.travis.yml

```diff
@@ -137,9 +144,7 @@ install:
    - which python
    - python --version
    - which pip
```

Comment:
Maybe along with `which pip` it would be good to run `pip list`, to see installed packages. It might be that there is an old version of numpy?
```
  ...
  File "/Users/travis/build/nest/nest-simulator/result/lib/python2.7/site-packages/nest/lib/hl_api_helper.py", line 35, in <module>
    import numpy
  File "/usr/local/lib/python2.7/site-packages/numpy/__init__.py", line 140, in <module>
    from . import _distributor_init
ImportError: cannot import name _distributor_init
```